src/app/pipes/percentageabs.pipe.ts

```diff
@@ -25,17 +25,21 @@ import BigNumber from 'bignumber.js'
 	name: 'percentageabs',
 })
 export class PercentageabsPipe implements PipeTransform {
-	transform(value_: number | BigNumber, max_: number | BigNumber, percentMode: boolean, preablePrct = ''): string {
-		const value = value_ instanceof BigNumber ? value_ : new BigNumber(value_)
-		const max = max_ instanceof BigNumber ? max_ : new BigNumber(max_)
-
+	transform(value_: number | BigNumber, percentage_: number | BigNumber, percentMode: boolean, preablePrct = ''): string {
 		if (percentMode) {
-			let percentValue = value.dividedBy(max).multipliedBy(100).decimalPlaces(1)
-			if (percentValue.toNumber() <= 0.1) {
-				percentValue = value.dividedBy(max).multipliedBy(100).decimalPlaces(3)
+			const percentage = percentage_ instanceof BigNumber ? percentage_ : new BigNumber(percentage_)
+			const percentageNumber = percentage.toNumber()
+			if (percentageNumber <= 0.1) {
+				return preablePrct + '0.1 %'
+			} else if (percentageNumber <= 1.0) {
+				return preablePrct + percentage.decimalPlaces(3).toString() + ' %'
+			} else if (percentageNumber <= 10.0) {
+				return preablePrct + percentage.decimalPlaces(2).toString() + ' %'
+			} else {
+				return preablePrct + percentage.decimalPlaces(1).toString() + ' %'
```

Comment:
Should be `<` and not `<=` in all cases, right? Also consider just setting a decimalPlaces variable in every [if-loop](http://www.if-schleife.de/) and returning `percentage.decimalPlaces(decimalPlaces).toString()` instead

Reply:
Made the according changes but kept the `<= 0.1` so that both 0.1 and everything below shows `0.1`.